templates/zerver/integrations.html

```diff
@@ -969,6 +969,124 @@
 
     </div>
 
+    <div id="gcal" class="integration-instructions">
+
+        <p>Get Google Calendar reminders in Zulip! This is a great way to see your reminders directly in
+        your Zulip feed.</p>
+
+        <h1>For Server Administrators:</h1>
```

Comment:
Shouldn't the application be created by the actual maintainers of the servers? For other applications, I wouldn't have to create a client secret file, and then give it access to my google account.